### PR TITLE
[WIP] added carousel

### PR DESCRIPTION
**Pull request by sudonitesh**
Don't know this template engine so added the same for loop with limit 4.  My previous try was to set time interval but that makes it unresponsive.

### CODE REVIEW

**Comment by sudonitesh** [comment]
@knrt10 how to lint?

**Comment by palash25** [comment]
@sudonitesh try googling code linting and look for a linter

**Comment by palash25** [comment]
Which linter did you use? Post the error log too.

**Comment by sudonitesh** [comment]
ok done . I had an extra closing tag.

**Comment by palash25** [comment]
This will help with squashing https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history

**Comment by knrt10** [comment]
@sudonitesh any updates?

**Comment by sudonitesh** [comment]
@knrt10  was that linting? can you please explain or some resource to see or tell me the errors?

File: index.html [hunk 1]
```diff
@@ -22,18 +22,117 @@ <h2 class="text-center">Explore</h2>
   </div>
 </div>
 
+<style>
+
```
Comment (knrt10) [inline index.html]: You see these spaces. When you lint a code these type of problems do not come. Try adding this in site [reference](https://github.com/Gizra/KnowledgeBase/wiki/Add-grunt-scss-lint-to-your-jekyll-site.) and then lint the code and update your PR.

File: index.html [hunk 2]
```diff
@@ -22,18 +22,117 @@ <h2 class="text-center">Explore</h2>
...
+    .pagination li.active, .pagination li:hover { background-color: #A7DDFF; }
+    
+    
+    </style>
+      
+    <script
+    src="https://code.jquery.com/jquery-3.3.1.js"
+    integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60="
+    crossorigin="anonymous"></script>
+    <script>
```
Comment (knrt10) [inline index.html]: Don't add script in same page. Create a different file for this

File: index.html [hunk 3]
```diff
@@ -22,18 +22,117 @@ <h2 class="text-center">Explore</h2>
...
     {% endfor %}
   </div>
+</div>
+<div class="portfolio-page" id="page-2" style="display:none;">
+    <div class="d-md-flex flex-wrap gutter flex-auto"> <!--six members here-->
+      {% assign contributors = site.contributors-app %}
+      {% for app in contributors limit : 4%}
+          {% include contributor.html app=app %}
+      {% endfor %}
+
```
Comment (knrt10) [inline index.html]: lint the code as I described above

**Comment by knrt10** [comment]
@sudonitesh you working on this?

**Comment by palash25** [comment]
Why was this closed ?

**Comment by sudonitesh** [comment]
@palash25  sorry..I made two pull request wrongly. so i closed this. but the code is same just added them in separate files.

**Comment by palash25** [comment]
whats the point of making a new PR if the code is the same. Also I think you have deleted your branch so I don't think if we can reopen this again.

Lets continue from the new one but this should not happen again :wink:

**Pull request closed without merging**